themes-default/slim/src/components/edit-show.vue

```diff
@@ -0,0 +1,363 @@
+                );
+            }).finally(() => {
+                // Re-enable the save button.
+                this.saving = false;
+            });
+        },
+        onChangeIgnoredWords(items) {
+            this.show.config.release.ignoredWords = items.map(item => item.value);
+        },
+        onChangeRequiredWords(items) {
+            this.show.config.release.requiredWords = items.map(item => item.value);
+        },
+        onChangeAliases(items) {
+            this.show.config.aliases = items.map(item => item.value);
+        },
+        onChangeReleaseGroupsAnime(items) {
+            this.show.config.release.whitelist = items.filter(item => item.memberOf === 'whitelist');
+            this.show.config.release.blacklist = items.filter(item => item.memberOf === 'blacklist');
+            this.show.config.release.allgroups = items.filter(item => item.memberOf === 'releasegroups');
+        },
+        updateLanguage(value) {
+            this.show.language = value;
+        }
+    },
+    // @TODO: Replace with Object spread (`...mapState`)
+    computed: {
+        ...mapState({
+            shows: state => state.shows.shows,
+            configLoaded: state => state.config.fanartBackground !== null,
+            config: state => state.config
+        }),
+        ...mapGetters({
+            show: 'getCurrentShow'
+        }),
+        indexer() {
+            return this.showIndexer || this.$route.query.indexername;
+        },
+        id() {
+            return this.showId || Number(this.$route.query.seriesid) || undefined;
+        },
+        // @TODO: Enable this once we remove this.series
+        // show() {
+        //     const { $store } = this;
+        //     return this.shows.length === 0 ? $store.defaults.show : this.shows.find(show => show.indexer === this.indexer && Number(show.id[show.indexer]) === Number(this.id));
+        // },
+        availableLanguages() {
+            if (this.config.indexers.config.main.validLanguages) {
+                return this.config.indexers.config.main.validLanguages.join(',');
+            }
+
+            return '';
+        },
+        combinedQualities() {
+            const { allowed, preferred } = this.show.config.qualities;
+            return combineQualities(allowed, preferred);
+        },
+        saveButton() {
+            return this.saving === false ? 'Save Changes' : 'Saving...';
+        },
+        showUrl() {
+            // @TODO: Change the URL generation to use `this.series`. Currently not possible because
+            // the values are not available at the time of app-link component creation.
+            return window.location.pathname.replace('editShow', 'displayShow') + window.location.search;
+        },
+        globalIgnored() {
+            return this.$store.state.search.filters.ignored.map(x => x.toLowerCase());
+        },
+        globalRequired() {
+            return this.$store.state.search.filters.required.map(x => x.toLowerCase());
```

Comment:
This was previously `this.$store.state.search.filters.ignored.map(x => x.toLowerCase());`